### DISCUSSION TITLE
Should I use MSFragger/fragpipe?

### DISCUSSION THREAD

**Discussion by ht-lau** [vdemichev/DiaNN#125] [Q&A]
Hi Vadim,

Thank you for the great tool. You have made DIA data analysis so much easier. 

I have tried the fragpipe > easypqp > DIA-NN GUI pipeline for ... no particular reason really... 

What difference(s) would I expect if I use DIA-NN only workflow?
And is there any situation you will recommend using the fragpipe workflow?

Thank you.

**Comment by vdemichev** [comment]
Hi,

By DIA-NN-only you mean library-free workflow? Yes, you can use it, sometimes it will be better, sometimes not. Can also try some publicly available spectral library - this is the third option. In general, the more samples in your experiment, the more they are heterogeneous, and the more peptides are identified in them on average, the better the performance of library-free in comparison to a spectral library. 

Best,
Vadim

**Reply by vdemichev** [reply]
But in many cases, especially for small-scale experiments, a good project specific spectral library (offline frac + DDA + FragPipe) is the best way to go.

**Comment by ht-lau** [comment]
Thank you for getting back to me. 

I tried the two workflows and I want to report back. I have 6 GPF DIA fractions. Using DIA-NN only yield ~ 3800 proteins; with DIA-Umpire > MSfragger, ~ 4200 proteins were identified. 

The MSfragger + DIA-NN workflow may take a little bit longer, but I think it is worth the time.

Thank you for the great tool again!

**Reply by vdemichev** [reply]
If you wish, you can share the DIA-NN log (that can be saved using the "Save log" button) and I will take a look.
